.github/workflows/generate-ai-training-ecs.yml

```diff
@@ -55,7 +55,7 @@ jobs:
           SUBNETS=$(if [ ${{ inputs.environment}} == "development" ]; then echo "subnet-034e5c652dbad09dd"; else echo "subnet-00b4e88c0ea178f3d"; fi)
           SECURITY_GROUPS=($(if [ ${{ inputs.environment}} == "development" ]; then echo "sg-09194f9a648baf082 sg-047498d4c7b2cedd8 sg-06d6458accc0ec5ed sg-0ace4338c75e5d3de"; 
             else echo "sg-02fd053fe7b4660ed sg-0e7119c4423f2c7b0 sg-066d0055caa7cbe90 sg-0c7fb74e11cb64ff8"; fi))
-          COMMAND="npm run start:generate-ai-training-set ${{ inputs.environment }} ${{ inputs.helpline-shortcodes }} tl-aselo-ai-${{ inputs.environment}}"
+          COMMAND="npm run start:generate-ai-training-set ${{ inputs.environment }} ${{ inputs.helpline-shortcodes }} tl-aselo-ai-${{ inputs.environment }}$(if [ ${{ inputs.environment }} == "production" ]; then echo "-us-east-1"; fi)"
```

Comment:
Is this change intentional in this branch?

Reply:
It was part of a different PR that got added when I merged master.